consensus/blake3pow/consensus.go

```diff
@@ -628,6 +628,7 @@ func (blake3pow *Blake3pow) Finalize(chain consensus.ChainHeaderReader, header *
 	}
 	header.Header().SetUTXORoot(state.UTXORoot())
 	header.Header().SetEVMRoot(state.IntermediateRoot(true))
+	header.Header().SetEtxSetRoot(state.ETXRoot())
```

Comment:
You will need to cherry pick df1e041c so that etxs are included in header views.